libs/application/templates/official-journal-of-iceland/src/hooks/useUpdateApplication.ts

```
@@ -51,6 +62,38 @@ export const useApplication = ({ applicationId }: OJOIUseApplicationParams) => {
+  const createApplication = async (
+    onComplete?: (data: { createApplication: Application }) => void,
+  ) => {
+    await createApplicationMutation({
+      variables: {
+        input: {
+          typeId: ApplicationTypes.OFFICIAL_JOURNAL_OF_ICELAND,
+          initialQuery: null,
+        },
+      },
+      onCompleted: (data) => {
+        onComplete && onComplete(data)
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)